andela/ah-orcas-frontend
# #162906138 Feedback Implementation

#### What does this PR do?
```This PR merges the feedback implementation branch to develop```
#### Description of Task to be completed?
```- change slug accessibility for both rating, like and disliking an article ```
```- add a label showing the author of an article on the homepage```
```- redirect a non logged in user trying to like/dislike article to login page```
```- add a back button on single article page```
#### How should this be manually tested?
- Clone this repository
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
```cd ah-orcas-frontend```
- Checkout to the appropriate branch
```git checkout ch-feedback-implementation-162906138```
- Install the required dependencies:
``` npm install```
- You should be able to: 
``` - be redirected to the login page whenever you try to like/dislike an article and you are not logged in```
 ```- see the username of the author of a given article```
#### Any background context you want to provide?
```Feedback implementation is essential in any project since it incorporates the needs of the target users into the project ```
#### What are the relevant pivotal tracker stories?
[#162906138](https://www.pivotaltracker.com/story/show/162906138)

#### Checklist:
